lib/ip.rb

```diff
@@ -0,0 +1,13 @@
+module IP
+  require 'ipaddr'
+  class IP
+    def initialize(ip_with_cidr)
```

Comment:
The constructor accepts not only a CIDR-notated IP but also just an exact IP address. So please comment it, since I couldn't understand the behavior from the argument's name.